Algorithms/Other/ParetoOptimization.cs

```diff
@@ -0,0 +1,79 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Algorithms.Other
+{
+    /// <summary>
+    /// Almost all real complex decision-making task is described by more than one criterion.
+    /// Therefore, the methods of multicriteria optimization are important. For a wide range
+    /// of tasks multicriteria optimization, described by some axioms of "reasonable"
+    /// behavior in the process of choosing from a set of possible solutions X, each set of
+    /// selected solutions Sel X should be contained in a set optimal for Pareto.
+    /// </summary>
+    public class ParetoOptimization
+    {
+        /// <summary>
+        /// Performs decision optimizations by using Paretor's optimization algorithm.
+        /// </summary>
+        /// <param name="matrix">Contains a collection of the criterias sets.</param>
+        public void Optimize(List<List<decimal>> matrix)
```

Comment:
Please don't mutate function arguments, just return your result from the function instead

Reply:
Fixed